src/Razor/src/Microsoft.AspNetCore.Razor.Language/Syntax/Generated/Syntax.xml.Syntax.Generated.cs

```diff
@@ -73,6 +73,9 @@ public RazorDocumentSyntax Update(RazorBlockSyntax document)
         if (document != Document)
         {
             var newNode = SyntaxFactory.RazorDocument(document);
+            var diagnostics = GetDiagnostics();
```

Comment:
We already do this for Green nodes https://github.com/aspnet/AspNetCore-Tooling/blob/master/src/Razor/src/Microsoft.AspNetCore.Razor.Language/Syntax/Generated/Syntax.xml.Internal.Generated.cs#L77